src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+
+function funcRead() {
```

Comment:
The function 'funcRead' is not properly named according to JavaScript conventions. Consider renaming it to something more descriptive, like 'copyFile'.